src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/GroovyMemoryLeakTest.java

```diff
@@ -0,0 +1,68 @@
+package org.jenkinsci.plugins.scriptsecurity.sandbox.groovy;
+
+import groovy.lang.MetaClass;
+import hudson.model.FreeStyleProject;
+import org.codehaus.groovy.reflection.ClassInfo;
+import org.junit.After;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.plugins.groovypostbuild.GroovyPostbuildRecorder;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.LoggerRule;
+import org.jvnet.hudson.test.MemoryAssert;
+
+import java.lang.ref.WeakReference;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+
+import static org.junit.Assert.assertFalse;
+
+/**
+ * @author Sam Van Oort
```

Comment:
Well…parts. :-) Never believe an `@author` tag and delete them when you find them.

Reply:
oops, okay removing that (autogenerated by IDE, sorry, definitely not trying to claim credit for your hard work)